k8s/airflow-operator/Makefile

```diff
@@ -0,0 +1,67 @@
+include ../vendor/marketplace-tools/app.Makefile
+include ../vendor/marketplace-tools/crd.Makefile
+include ../vendor/marketplace-tools/gcloud.Makefile
+include ../vendor/marketplace-tools/marketplace.Makefile
+include ../vendor/marketplace-tools/ubbagent.Makefile
+include ../vendor/marketplace-tools/var.Makefile
+
+
+TAG ?= latest
+$(info ---- TAG = $(TAG))
+
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/airflow-operator/deployer:$(TAG)
+NAME ?= airflow-operator-1
+APP_PARAMETERS ?= { \
+  "name": "$(NAME)", \
+  "namespace": "$(NAMESPACE)" \
+}
+TESTER_IMAGE ?= $(REGISTRY)/airflow-operator/tester:$(TAG)
+APP_TEST_PARAMETERS ?= { \
+  "testerImage": "$(TESTER_IMAGE)" \
+}
+
+
+app/build:: .build/airflow-operator/deployer \
+            .build/airflow-operator/airflow-operator \
+            .build/airflow-operator/tester
+
+
+.build/airflow-operator: | .build
+	mkdir -p "$@"
+
+
+.build/airflow-operator/deployer: deployer/* \
+                                manifest/* \
+                                schema.yaml \
```

Comment:
Good point, I think other apps are missing the dependency on `schema.yaml` for their deployers.

Reply:
ack